### PR TITLE
Fix javadoc generation

### PR DESCRIPTION
**Pull request by rPraml**
Hello Rob,
I noticed, that there are a lot of javadoc warnings during a maven build.
I do some investigation and noted, that it is currently impossible to use an `@` char in an `@{code ...}` block.

This is clearly a Javadoc-Bug, but it seems to exist since a long time, so I think, there will be no fix in the future (See: https://stackoverflow.com/questions/23812761/trying-to-escape-the-symbol-in-a-code-block-within-a-javadoc-comment-with)

To fix invalid javadoc (see https://ebean.io/apidoc/12/io/ebean/RowMapper.html) I changed them to `<code>`
There is also a missing dependency in `avaje-lang`, which causes `unknown enum constant When,MAYBE` warning.

Unfortunately, there are still a lot of files with invalid javadoc (mostly generated by xjc) which are not part of this PR

### CODE REVIEW

File: ebean-api/pom.xml [hunk 1]
```diff
@@ -39,6 +39,12 @@
       <artifactId>avaje-lang</artifactId>
       <version>1.0</version>
     </dependency>
+    
+    <dependency>
+      <groupId>io.avaje</groupId>
+      <artifactId>avaje-jsr305</artifactId>
+      <version>1.2</version>
+    </dependency>
```
Comment (rPraml) [inline ebean-api/pom.xml:47]: This dependency has provided scope in avaje-lang.
@rbygrave please check, if we really want to have this dependency here or if it should be fixed in avaje-lang instead

**Comment by rbygrave** [comment]
The javadoc bug is fixed in 15.

**Comment by rbygrave** [comment]
No, we don't want these changes as it makes the example code in the javadoc harder to read.

If you bump you JDK to 15 or 17 then you won't see the javadoc warnings so that is what I'm suggesting you do or just ignore the warnings.

**Comment by rPraml** [comment]
Thanks, I ignore the errors
(We recently made the jump to 11 now. It will probably be a while before we can go to 15 in our company. ;) )

**Comment by rbygrave** [comment]
Ha ha. I note that there are some pretty impressive improvements to GC in 18 which has just gone GA :)

**Pull request closed without merging**